google/node_config.go

```diff
@@ -128,6 +150,22 @@ func expandNodeConfig(v interface{}) *container.NodeConfig {
 		nc.MachineType = v.(string)
 	}
 
+	if v, ok := nodeConfig["guest_accelerator"]; ok {
+		accels := v.([]interface{})
+		guestAccelerators := make([]*container.AcceleratorConfig, 0, len(accels))
+		for _, raw := range accels {
+			data := raw.(map[string]interface{})
```

Comment:
It seems like you could make accels a []map[string]interface{}, which would mean you'd only have to do one of these casts.

Reply:
I lazily lifted this section of code from at https://github.com/terraform-providers/terraform-provider-google/blob/master/google/resource_compute_instance.go#L1384, so it may be better to maintain consistency

Reply:
Making accels a `[]map[string]interface{}` leads to an error:
```
panic: interface conversion: interface {} is []interface {}, not []map[string]interface {}

goroutine 297 [running]:
github.com/terraform-providers/terraform-provider-google/google.expandNodeConfig(0x16797c0, 0xc4205361c0, 0xb)
	/home/i056593/Code/go/src/github.com/terraform-providers/terraform-provider-google/google/node_config.go:154 +0x12e0
```

Reply:
(Side note: Line 154 is basically copied from https://github.com/terraform-providers/terraform-provider-google/blob/master/google/resource_compute_instance_template.go#L504)

Reply:
Ah, all right, that makes sense.